types/prismjs/index.d.ts

```diff
@@ -148,6 +175,41 @@ export namespace util {
 
     /** Deep clone a language definition (e.g. to extend it) */
     function clone<T>(o: T): T;
+
+    /**
+     * Returns the Prism language of the given element set by a `language-xxxx` or `lang-xxxx` class.
+     *
+     * If no language is set for the element or the element is `null` or `undefined`, `none` will be returned.
+     */
+    function getLanguage(element: Element): string;
```

Comment:
No. These are internal utility functions that might change at any time. They are only exported here because some plugins need these functions.

They are not part of the public API and should not be used by others.

The other util function are technically internal as well, but they are quite old and have existed since before we drew a clear line between public and private API. They are documented here not because people should use them, but because we don't want to break the code of people that might rely on it.

Reply:
thanks. I will remove that